Sources/URLUtils.swift

```diff
@@ -13,7 +13,7 @@ import AEPServices
 import Foundation
 
 /// Provides functions to append visitor information to a URL
-enum URLAppender {
+enum URLUtils {
 
     /// Helper function to generate url variables in format acceptable by the AEP web SDK (Alloy)
```

Comment:
let's keep this generic as it applies for the Visitor library also, e.g. "acceptable by the Adobe Web SDKs".